code/modules/client/preferences.dm

```diff
@@ -276,6 +279,15 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 
 			return TRUE
 
+		if ("test_tts")
+			if(!COOLDOWN_FINISHED(src, tts_test_cooldown))
+				to_chat(usr, "TTS preview on cooldown.")
```

Comment:
This is terrible user feedback because you're not looking in the chat when you have a big preferences menu in your face. Have the test button be disabled on the client when in cooldown, and don't bother with anything here.

Reply:
I'm going to remove this message entirely. The cooldown length is short enough that visual feedback isn't really necessary.